CHANGES.md

```diff
@@ -1,4 +1,5 @@
 # UNRELEASED
+- [FIXED] Case where if the username, password, and url existed during client creation the url was missing the required credentials.
```

Comment:
This is quite implementation specific, perhaps we can make it more user centric?
Maybe something like this:
```markdown
- [FIXED] Case where `username` and `password` options were not used if a `url` was supplied.
```
wdyt?

Reply:
Yea, sounds better.